hcaptcha/api/hcaptcha.api

```diff
@@ -0,0 +1,103 @@
+public final class com/stripe/hcaptcha/BuildConfig {
+	public static final field BUILD_TYPE Ljava/lang/String;
+	public static final field DEBUG Z
+	public static final field LIBRARY_PACKAGE_NAME Ljava/lang/String;
+	public fun <init> ()V
+}
+
+public final class com/stripe/hcaptcha/HCaptcha$Companion {
```

Comment:
You should be able to remove this by adding a restrict to at the companion level.